lte/gateway/deploy/roles/agw_docker/templates/env.j2

```diff
@@ -11,7 +11,7 @@
 # limitations under the License
 
 COMPOSE_PROJECT_NAME=agw
-DOCKER_REGISTRY={{ docker_registry }}
+DOCKER_REGISTRY={{ docker_registry }}/
```

Comment:
that feels like a rather random place to add that slash. Wouldn't it be  better to do that in ansible or add the slash to the variable?

Reply:
Then, if any user will be changing registry, they will have to remember to add the slash. Because we need it all the time, it's not something that should be changed with a variable.

Reply:
I'm also a bit confused by this change. Wouldn't it be better to just add a comment that the slash is needed where the user sets the `docker_registry` variable? Also, in case the user doesn't set a custom registry, would it be a problem that `DOCKER_REGISTRY` is now set to `/`?

Reply:
Good point, okay, I will move the slash to the variable.

Reply:
@maxhbr @sebathomas I have updated the PR.